Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TextToolbar/TextToolbarPage.xaml.cs

```diff
@@ -0,0 +1,107 @@
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.Toolkit.Uwp.UI.Controls;
+using Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarButtons;
+using Microsoft.Toolkit.Uwp.UI.Controls.TextToolbarFormats.MarkDown;
+using Windows.System;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Navigation;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    public sealed partial class TextToolbarPage
+    {
+        public TextToolbarPage()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnNavigatedTo(NavigationEventArgs e)
+        {
+            base.OnNavigatedTo(e);
+
+            Shell.Current.RegisterNewCommand("Add/Remove Bold Button", (sender, args) =>
+            {
+                var button = Toolbar.GetDefaultButton(ButtonType.Bold);
+                button.Visibility = button.Visibility == Windows.UI.Xaml.Visibility.Visible ? Windows.UI.Xaml.Visibility.Collapsed : Windows.UI.Xaml.Visibility.Visible;
+            });
+
+            Shell.Current.RegisterNewCommand("Add Custom Button", (sender, args) =>
+            {
+                AddCustomButton();
+            });
+        }
+
+        private int DemoCounter { get; set; } = 0;
+
+        private void AddCustomButton()
+        {
+            string demoText = "Demo";
+            demoText = DemoCounter > 0 ? demoText + DemoCounter : demoText;
+
+            int keycode = (int)VirtualKey.Number0 + DemoCounter;
+            VirtualKey? shortcut = null;
+
+            if (keycode <= (int)VirtualKey.Number9)
+            {
+                shortcut = (VirtualKey)keycode;
+            }
+
+            DemoCounter++;
+
+            var demoButton = new ToolbarButton
+            {
+                Icon = new SymbolIcon { Symbol = Symbol.ReportHacked },
+                ToolTip = demoText,
+                ShortcutKey = shortcut,
+                Activation = (b) =>
+                {
+                    var md = Toolbar.Formatter as MarkDownFormatter;
+                    if (md != null)
+                    {
+                        md.SetSelection($"[{demoText}]", $"[/{demoText}]");
+                    }
+                    else
+                    {
+                        Toolbar.Formatter.Selected.Text = $"This was filled by {demoText} button ";
+                    }
+                }
+            };
+
+            Toolbar.CustomButtons.Add(demoButton);
```

Comment:
If the toolbar is wider than the parent hosting it, it would be great to have something that indicates there is more.